moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -42,12 +42,12 @@
 #include <vector>
 #include <map>
 #include <memory>
-#include <ros/ros.h>
+#include "rclcpp/rclcpp.hpp"
 #include <moveit/profiler/profiler.h>
 
 namespace kinematics_plugin_loader
 {
-static const std::string LOGNAME = "kinematics_plugin_loader";
+rclcpp::Logger LOGGER_KINEMATICS_PLUGIN_LOADER = rclcpp::get_logger("kinematics_plugin_loader");
```

Comment:
Please apply the logging best practices https://github.com/ros-planning/moveit2/blob/b96a0259eb8d35364ab7d52f26fd50cdd45ee513/doc/MIGRATION_GUIDE.md